src/main/java/racingcar/Game.java

```diff
@@ -0,0 +1,31 @@
+package racingcar;
+
+import java.util.stream.Collectors;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.movable.MovableStrategy;
+import racingcar.view.OutputView;
+
+public class Game {
+
+    private final Cars cars;
+    private final String WINNER_DELIMITER = ", ";
```

Comment:
static이 누락된 것 같네요 :) 

```suggestion
    private static final String WINNER_DELIMITER = ", ";
```